### ISSUE TITLE
Lack of Serialization in TypeHandler interface.

### ISSUE THREAD

**Issue by szczurmys** [szczurmys/ignite-cassandra-store#2]
Lack of Serialization in TypeHandler interface.

TypeHandler cause exception:
```
Caused by: java.io.NotSerializableException: com.github.szczurmys.ignite.cache.store.cassandra.handler.StringTypeHandler
```
Related issue #1

**Comment by szczurmys** [comment]
Fixed in 2.5.1 and 2.6.1

**Comment by rambai** [comment]
Thank you very much for solving issue so fast!
